extensions.md

```diff
@@ -26,26 +26,25 @@ as additional parameters for search through either **STAC API - Item Search** or
-| [Sort](https://github.com/stac-api-extensions/sort/blob/main/README.md)                                 | [STAC API - Item Search](item-search/) request                                                             | Adds Parameter to control sorting of returns results.                                                                                       | Candidate  |
-| [Transaction](https://github.com/stac-api-extensions/transaction/blob/main/README.md)                   | [STAC API - Features](ogcapi-features) POST on `/items` endpoint, DELETE/PUT on `/items/{itemId}` endpoint | Adds PUT and DELETE endpoints for the creation, editing, and deleting of Item objects.                                                      | Candidate  |
-| [Items and Collections API Version](https://github.com/stac-api-extensions/version/blob/main/README.md) | [STAC API - Features](ogcapi-features) on `/items` endpoint                                                | Adds GET versions resource to Collection and Item endpoints and provides semantics for a versioning scheme for Collection and Item objects. | *Proposal* |
-| [Query](https://github.com/stac-api-extensions/query/blob/main/README.md)                               | [STAC API - Item Search](item-search/) request                                                             | Adds parameter to search Item and Collection properties.                                                                                    | Candidate  |
-| [Children](https://github.com/stac-api-extensions/children/blob/main/README.md)                         | [STAC API - Core](core/)                                                                                   | Defines advertisement of the children (child catalogs or child collections) a catalog contains.                                             | Proposal   |
+| Extension Name                                                                        | Scope                                                                             | Description                                                                                          | Maturity  |
+| ------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- | --------- |
+| [Fields](https://github.com/stac-api-extensions/fields/blob/main/README.md)           | [STAC API - Item Search](item-search/) and [STAC API - Features](ogcapi-features) | Adds parameter to control which fields are returned in the response.                                 | Candidate |
```

Comment:
Learning from stac-spec, I think we should publish a list such as https://stac-extensions.github.io/ and then link to it instead of maintaining the extensions in this repo. It just gets outdated too quickly.

Reply:
Sounds good. I've filed this issue to do that work https://github.com/radiantearth/stac-api-spec/issues/372

I'd still like to merge this PR. and then move it after that.

Reply:
Okay, good for me. Should we already add Collection Search and Language here?

Reply:
Yes, I'll add that after I create the new webpage. Also Aggregation.